src/bindings/python/src/openvino/frontend/pytorch/torchdynamo/backend_utils.py

```diff
@@ -83,3 +83,10 @@ def _is_testing(options) -> Optional[Any]:
             return True
     return False
 
+def _executorch(options) -> Optional[Any]:
```

Comment:
If executorch is added in options, this will also be available to torch.compile users and may break the flow if the option is set to True as ExecuTorchPythonDecoder will be used. Instead, can we add it as a kwarg to openvino_compile and set it to False by default? This can be set to True in Executorch openvino backend. That way it is not visible to the user. Please let me know your thoughts.

Reply:
I see, I was trying not to increase the number of parameters for openvino_compile at first but it makes sense not to expose this feature to users. I think it is a better idea to pass this as kwarg. I will update accordingly.